Instantiate StandardType<std::tuple<>>

```
/opt/civet/build_0/moose/scripts/../libmesh/installed/include/timpi/standard_type.h: In instantiation of 'TIMPI::StandardType<std::tuple<_Tps ...> >::StandardType(const TIMPI::StandardType<std::tuple<_Tps ...> >&) [with Types = {long unsigned int, unsigned int, unsigned int}]':
/opt/civet/build_0/moose/scripts/../libmesh/installed/include/timpi/standard_type.h:101:10:   required from 'TIMPI::StandardType<T> TIMPI::build_standard_type(const T*) [with T = std::tuple<long unsigned int, unsigned int, unsigned int>]'
/opt/civet/build_0/moose/scripts/../libmesh/installed/include/timpi/parallel_sync.h:180:34:   required from 'void TIMPI::push_parallel_vector_data(const TIMPI::Communicator&, const MapToVectors&, const ActionFunctor&) [with MapToVectors = std::unordered_map<unsigned int, std::vector<std::tuple<long unsigned int, unsigned int, unsigned int> > >; ActionFunctor = RayTracingMeshOutput::buildIDMap()::<lambda(libMesh::processor_id_type, const std::vector<std::tuple<long unsigned int, unsigned int, unsigned int> >&)>]'
/opt/civet/build_0/moose/modules/ray_tracing/src/outputs/RayTracingMeshOutput.C:264:67:   required from here
/opt/civet/build_0/moose/scripts/../libmesh/installed/include/timpi/standard_type.h:410:3: error: base class 'class TIMPI::DataType' should be explicitly initialized in the copy constructor [-Werror=extra]
   StandardType(const StandardType<std::tuple<Types...>> & timpi_mpi_var(t))
```

--- a/src/parallel/include/timpi/standard_type.h
+++ b/src/parallel/include/timpi/standard_type.h
@@ -363,7 +363,9 @@ class StandardType<std::tuple<Types...>> : public DataType
 {
 public:
   explicit
-  StandardType(const std::tuple<Types...> * example = nullptr) {
+  StandardType(const std::tuple<Types...> * example = nullptr)
+    : DataType()
+  {
     // We need an example for MPI_Address to use
     static const std::tuple<Types...> t;
     if (!example)
@@ -416,6 +418,7 @@ public:
   }
 
   StandardType(const StandardType<std::tuple<Types...>> & timpi_mpi_var(t))
+    : DataType()
   {
     timpi_call_mpi
       (MPI_Type_dup (t._datatype, &_datatype));

